pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -134,6 +135,12 @@ func (r *MigPlan) GetRefResources(client k8sclient.Client) (*PlanResources, erro
 	return resources, nil
 }
 
+// Set the MigPlan Status to closed
+func (r *MigPlan) SetClosed() error {
```

Comment:
Why does `SetClosed()` return an _error_?

Reply:
I wondered about whether to do this or have no return. I left the error return for consistency with similar methods -- we have others that return an error but there's no actual error returns in the current implementation. I can get rid of the return, though, if there's no need to keep the method signatures consistent for this sort of thing.

Reply:
I appreciate consistency.  
The _other_ methods returning an error are also passed a client.  The _setter_ provides a nice abstraction from the Spec but don't think it's sufficiently complex to be passed a client.  I recommend removing the _error_ return value here.

Reply:
OK. That works for me.